Source/WebCore/Modules/mediastream/PeerConnectionBackend.cpp

```diff
@@ -145,7 +197,7 @@ void PeerConnectionBackend::createOfferSucceeded(String&& sdp)
 void PeerConnectionBackend::createOfferFailed(Exception&& exception)
 {
     ASSERT(isMainThread());
-    ALWAYS_LOG(LOGIDENTIFIER, "Create offer failed:", exception.message());
+    ALWAYS_LOG(LOGIDENTIFIER, makeString("Create offer failed:"_s, exception.message()));
```

Comment:
It is less efficient to have the makeString call if we do not log. I would tend to keep it as is.